locales/de.json

```diff
@@ -1,574 +1,911 @@
+    },
+    "landmark-no-duplicate-contentinfo": {
+      "description": "",
+      "help": "Das Dokument sollte höchstens eine contentinfo landmark enthalten."
+    },
+    "landmark-no-duplicate-main": {
+      "description": "Stellt sicher, dass das Dokument höchstens eine main-Landmark besitzt.",
+      "help": "Das Dokument sollte nur eine main-Landmark besitzen."
+    },
+    "landmark-one-main": {
+      "description": "",
+      "help": "Seite muss eine main landmark enthalten."
+    },
+    "landmark-unique": {
+      "help": "Stellt sicher, dass landmarks einzigartig sind.",
+      "description": "Landmarks müssen eine einzigartige role oder role/label/title Kombination (bzw. zugänglicher Name) besitzen."
+    },
+    "link-in-text-block": {
+      "description": "",
+      "help": "Links müssen vom umgebenden Text auf eine Weise unterschieden werden können, die nicht allein auf Farbe beruht."
+    },
+    "link-name": {
+      "description": "",
+      "help": "Links müssen wahrnehmbaren Text enthalten."
+    },
+    "list": {
+      "description": "",
+      "help": "<ul>- und <ol>-Elemente dürfen unmittelbar nur <li>-, <script>- oder <template>-Elemente enthalten."
+    },
+    "listitem": {
+      "description": "",
+      "help": "<li>-Elemente müssen in einem <ul>- oder <ol>-Element enthalten sein."
+    },
+    "marquee": {
+      "description": "",
+      "help": "<marquee>-Elemente sind veraltet und dürfen nicht verwendet werden."
+    },
+    "meta-refresh": {
+      "description": "",
+      "help": "Eine zeitgesteuerte Aktualisierung (refresh) sollte nicht verwendet werden."
+    },
+    "meta-viewport-large": {
+      "description": "",
+      "help": "Benutzer sollten in der Lage sein, den Text um bis zu 500% vergrößern und skalieren zu können."
+    },
+    "meta-viewport": {
+      "description": "",
+      "help": "Zoomen und Skalieren darf nicht deaktiviert werden."
+    },
+    "no-autoplay-audio": {
+      "description": "Stellt sicher, dass <video> oder <audio> Elemente keine Töne automatisch abspielen für mehr als 3 Sekunden (autoplay) ohne eine Möglichkeit dies zu stoppen.",
+      "help": "<video> oder <audio> Elemente geben keine Töne automatisch aus."
+    },
+    "object-alt": {
+      "description": "",
+      "help": "<object>-Elemente müssen einen Alternativtext besitzen."
+    },
+    "p-as-heading": {
+      "description": "",
+      "help": "Die Schriftschnitte bold und italic sowie die Schriftgröße dürfen nicht verwendet werden, um <p>-Elemente wie Überschriften zu formatieren."
+    },
+    "page-has-heading-one": {
+      "description": "",
+      "help": "Die Seite muss eine Überschrift erster Ebene enthalten."
+    },
+    "presentation-role-conflict": {
+      "description": "Markiert Elemente welche eine Rolle besitzen, welche none oder presentation ist und welche eine Rollenauflösung benötigen.",
+      "help": "Elemente mit Rolle none oder presentation sollten markiert werden."
```

Comment:
Change this to `Elemente mit Rolle none oder presentation dürfen nich in konflikt stehen mit anderen Rollen`